scripts/docker-compose.yml

```diff
@@ -47,3 +47,9 @@ aerospike:
     image: aerospike/aerospike-server
     ports:
         - "3000:3000"
+
+nsq:
+    image: nsqio/nsq
+    ports:
+        - "4150:4150"
+        - "4151:4151"
```

Comment:
4150 is an TCP listener port, 4151 is for HTTP even though the plugin isnt using that feature should I remove it?